tests/check/check_h26xsigned_auth.c

```diff
@@ -2094,47 +2089,28 @@ START_TEST(_is_start_of_stream_byte)
   ck_assert_int_eq(sv_rc, SV_OK);
   sv_rc = signed_video_set_authenticity_level(sv, settings[_i].auth_level);
   ck_assert_int_eq(sv_rc, SV_OK);
-  sv_rc = signed_video_set_is_start_stream(sv, true);
-  ck_assert_int_eq(sv_rc, SV_OK);
 
   // Add I-frame for signing and get SEI frame.
   sv_rc = signed_video_add_nalu_for_signing_with_timestamp(
       sv, i_1_nalu->data, i_1_nalu->data_size, &g_testTimestamp);
   ck_assert_int_eq(sv_rc, SV_OK);
-  // Check the first I frame
+  // Check the I frame
   ck_assert_int_eq(
-      signed_video_is_start_of_stream(i_1_nalu->data, i_1_nalu->data_size, codec), false);
+      signed_video_is_start_of_stream_sei(sv, i_1_nalu->data, i_1_nalu->data_size), false);
```

Comment:
You should remove this entire test. It adds no value. It is better to add this type of line in an appropriate function in signed_video_helpers.c